lib/src/geo/crs/crs.dart

```diff
@@ -132,7 +131,7 @@ class Epsg3857 extends Earth {
 
 // Epsg3857 seems to have latitude limits. https://epsg.io/3857
 //@override
-//Tuple2<double, double> get wrapLat => const Tuple2(-85.06, 85.06);
+//(double, double get wrapLat => const Tuple2(-85.06, 85.06);
```

Comment:
Should be:
```dart
(double, double) get wrapLat => const (-85.06, 85.06);
```

Reply:
https://github.com/fleaflet/flutter_map/pull/1517/commits/2bff4b7a32fbf9a41358e994449388de4ac25b19